### ISSUE TITLE
Indent false negative

### ISSUE THREAD

**Issue by feross** [eslint/eslint#7637]
**Tell us about your environment**

* **ESLint Version:** 3.10.2
* **Node Version:** 7.1.0

**What parser (default, Babel-ESLint, etc.) are you using?** default

**Please show your full configuration:**

[eslint-config-standard 6.2.1](https://github.com/feross/eslint-config-standard/blob/c879dfb71e03739a45e980687f9a28c94b4a95d2/eslintrc.json)

Specifically:

```json
{
  "indent": [2, 2, { "SwitchCase": 1 }]
}
```

**What did you do? Please include the actual source code causing the issue.**

```js
export default db => {
  return db.collection('option').insertMany([
      { },
      { }
  ])
}
```

**What did you expect to happen?**

Error on lines 3 and 4 for invalid indentation.

**What actually happened? Please include the actual, raw output from ESLint.**

No error.

Original issue: https://github.com/feross/standard/issues/688

**Comment by not-an-aardvark** [comment]
Thanks for the report, but I think this is a duplicate of https://github.com/eslint/eslint/issues/7242.

**Comment by feross** [comment]
No problem. I'll close.
